src/main/scala/diplomacy/Nodes.scala

```diff
@@ -76,10 +76,17 @@ abstract class BaseNode(implicit val valName: ValName)
   protected[diplomacy] def instantiate(): Seq[Dangle]
   protected[diplomacy] def finishInstantiate(): Unit
 
-  def name = scope.map(_.name).getOrElse("TOP") + "." + valName.name
-  def omitGraphML = outputs.isEmpty && inputs.isEmpty
+  def name: String = scope.map(_.name).getOrElse("TOP") + "." + valName.name
+  def omitGraphML: Boolean = outputs.isEmpty && inputs.isEmpty
   lazy val nodedebugstring: String = ""
 
+  /** Mark whether the node represents an identity function over the parameters moving through it.
+    *
+    * This information may be used to elide monitors or inline the parent module,
+    * since it indicates that the circuitry generated by this node should have no consequence.
```

Comment:
This comment contradicts itself. There are plenty of circuits that have identity for parameters but have a consequential circuit.